ractor.rb

```diff
@@ -148,10 +154,10 @@
 #
 # Besides frozen objects, there are shareable objects. Class and Module objects are shareable so
 # the Class/Module definitions are shared between ractors. Ractor objects are also shareable objects.
-# All operations for the shareable mutable objects are thread-safe, so the thread-safety property
+# All operations on shareable mutable objects are thread-safe, so the thread-safety property
 # will be kept. We can not define mutable shareable objects in Ruby, but C extensions can introduce them.
 #
-# It is prohibited to access instance variables of mutable shareable objects (especially Modules and classes)
```

Comment:
Why remove "mutable" here?

Reply:
It's prohibited to access instance variables of shareable objects, that's all there is to it 😄

Reply:
I made this section more clear because it was confusing. You can access instance variables of shareable objects in other ractors but the values of the variables must be shareable.